stdlib/public/core/CompilerProtocols.swift

```diff
@@ -194,10 +194,46 @@ public protocol _ExpressibleByBuiltinIntegerLiteral {
   init(_builtinIntegerLiteral value: _MaxBuiltinIntegerType)
 }
 
-/// Conforming types can be initialized with integer literals.
+/// A type that can be initialized with an integer literal.
```

Comment:
Good call, I have another branch landing on master soon; I'll move it over there.